### PR TITLE
[fix](doc)description of stream-load

### PR DESCRIPTION
**Pull request by yz-jayhua**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: docs/zh-CN/docs/sql-manual/sql-reference/Data-Manipulation-Statements/Load/STREAM-LOAD.md [hunk 1]
```diff
@@ -123,7 +123,7 @@ curl --location-trusted -u user:passwd [-H ""...] -T data.file -XPUT http://fe_h
     
 16. merge_type: 数据的合并类型，一共支持三种类型APPEND、DELETE、MERGE 其中，APPEND是默认值，表示这批数据全部需要追加到现有数据中，DELETE 表示删除与这批数据key相同的所有行，MERGE 语义 需要与delete 条件联合使用，表示满足delete 条件的数据按照DELETE 语义处理其余的按照APPEND 语义处理， 示例：`-H "merge_type: MERGE" -H "delete: flag=1"`
 
-17. delete: 仅在 MERGE下有意义， 表示数据的删除条件
+17. delete: 仅在 MERGE下有意义， 表示数据的删除条件<br>
```
Comment (JNSimba) [inline docs/zh-CN/docs/sql-manual/sql-reference/Data-Manipulation-Statements/Load/STREAM-LOAD.md:126]: not use <br>

Comment (yz-jayhua) [reply docs/zh-CN/docs/sql-manual/sql-reference/Data-Manipulation-Statements/Load/STREAM-LOAD.md]: > not use `<br>`

Thx

**Review by zy-kkk — APPROVED** [review]
LGTM

**Comment by zy-kkk** [comment]
run buildall

**Review by JNSimba — APPROVED** [review]
LGTM

**Pull request merged**